internal/pkg/template/templates/environment/partials/cdn-resources.yml

```diff
@@ -1,10 +1,58 @@
+{{- if or .CDNConfig.ViewerCertificate .DelegateDNS}}
```

Comment:
Can we rename `ViewerCertificate` to be `ImportedCertificate`? since `ViewerCertificate` is too CF specific. Plus without `.CDNConfig.ViewerCertificate` doesn't even mean we don't have `ViewerCertificate` for CF. Instead it just means we don't have any imported cert for CF/CDN.

Reply:
i'm wondering if that could cause issues, based on https://github.com/aws/copilot-cli/pull/3881#discussion_r943692647. If they import a cert to their public lb without setting the cdn cert, we would be still be setting the CF `Aliases` field without any certificates for those aliases

Reply:
> If they import a cert to their public lb without setting the cdn cert

We'll error out in this condition i think. I don't understand why renaming will cause any issue 🤔

Reply:
i misunderstood what you were asking 😊 i renamed it!